test/graph-model-test.js

```diff
@@ -23,6 +23,13 @@ suite('graph-model-test', () => {
     assert.equal(model.beanType, attrs.beanType);
   });
 
+  test('it creates an object with enumerable keys', () => {
+
+    const model = new GraphModel(attrs);
+
+    assert.deepEqual(Object.keys(model), ['beans', 'beanType']);
```

Comment:
attrs is non-enumerable by default. if somehow it does become enumerable this existing test will catch that.